agreements/membership/members.yaml

```diff
@@ -43,6 +45,16 @@ role:
       - coc
       - egc
 
+  advisor:
+    title: "Advisory advisory-board: Membership Agreement"
```

Comment:
```suggestion
    title: "Advisor advisory-board: Membership Agreement"
```

Feel free to ignore, but "Advisory advisory" just sounded odd to me

Reply:
good catch. Typo for sure